.serena/memories/suggested_commands.md

```
@@ -0,0 +1,84 @@
+
+# Search in files
+Select-String -Path "*.ts" -Pattern "pattern"
+
+# Git operations
+git status
+git log --oneline
+git diff
+```
+
+## Local Testing URLs
+- Gallery: http://localhost:3000/zukan
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)